debezium-connector-mysql/src/test/java/io/debezium/connector/mysql/MySqlParserIT.java

```diff
@@ -0,0 +1,134 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.debezium.connector.mysql;
+
+import static org.fest.assertions.Assertions.assertThat;
+
+import java.sql.SQLException;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.testcontainers.containers.BindMode;
+import org.testcontainers.containers.MySQLContainer;
+import org.testcontainers.containers.output.Slf4jLogConsumer;
+import org.testcontainers.utility.DockerImageName;
+
+import io.debezium.config.CommonConnectorConfig;
+import io.debezium.config.Configuration;
+import io.debezium.embedded.AbstractConnectorTest;
+import io.debezium.jdbc.JdbcConfiguration;
+import io.debezium.jdbc.JdbcConnection;
+import io.debezium.util.Testing;
+
+/**
+ * Integration test for {@link MySqlConnector} using Testcontainers infrastructure for testing column constraints supported in MySQL 8.0.x.
+ */
+
+public class MySqlParserIT extends AbstractConnectorTest {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(MySqlParserIT.class);
+
+    private static final String MYSQL_IMAGE = "debezium/example-mysql:1.7";
```

Comment:
Can you log a follow-up issue to replace this hard-coded version with the code for looking up the latest stable version? We can only do this once 1.7 has been finalized though, IIRC. Thx!